hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/configuration/FlinkOptions.java

```diff
@@ -168,12 +169,11 @@ private FlinkOptions() {
   public static final ConfigOption<String> SUPPLEMENTAL_LOGGING_MODE = ConfigOptions
       .key("cdc.supplemental.logging.mode")
       .stringType()
-      .defaultValue("cdc_data_before_after") // default record all the change log images
+      .defaultValue(op_key_only.name())
       .withFallbackKeys(HoodieTableConfig.CDC_SUPPLEMENTAL_LOGGING_MODE.key())
```

Comment:
Let's keep the default value of Flink as `data_before_after`, there are some issues maybe for the other 2 modes because Flink can trigger eager flush, the intermediate data file may cause changes loss, we can switch to other modes when I fix that.

Reply:
ok got it

Reply:
@danny0405 do you have jira to track this work? i'd link it from the code as todo

Reply:
Just create one: https://issues.apache.org/jira/browse/HUDI-5636